src/drop-down-menu.jsx

```diff
@@ -211,11 +211,13 @@ let DropDownMenu = React.createClass({
             autoWidth={this.props.autoWidth}
             selectedIndex={selectedIndex}
             menuItems={menuItems}
+            style={styles.menu}
             menuItemStyle={this.mergeAndPrefix(styles.menuItem, this.props.menuItemStyle)}
             hideable={true}
             visible={this.state.open}
             onRequestClose={this._onMenuRequestClose}
             onItemTap={this._onMenuItemClick} />
+          {this.state.open && <div style={styles.overlay} onTouchTap={this._onTouchTapOverlay} />}
```

Comment:
@oliviertassinari Very nice solution! Just have a small naming change on this line. For event handling I usually use: `_handleOverlayTouchTap`.